force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -175,12 +228,14 @@ public with sharing class ClassesObjectsMethods {
      * @return The current date in the current user locale format.
      */
     public static String getCurrentDate() {
-        // Get the todays date
-
-        // Format the current date
-
-        // Return the result
-        return null; // Replace null with the variable you used to store the result
+     // Get the current date
+     Datetime now = Datetime.now();
+
+     // Format the current date
+     String dateString = now.format('dd.MM.yyyy');
+  
+     // Return the result
+     return dateString;
```

Comment:
I think this is the one I saw fail a unit test. Try again and use the Date class not Date Time.